dev/start.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+set -ex
+
+function clean {
+echo 'Removing test environment'
+echo '---'
+docker-compose down -v
+rm -rf inventory.tmp
+}
+
+# normalises project name by filtering non alphanumeric characters and transforming to lowercase
+declare -x COMPOSE_PROJECT_NAME
+
+COMPOSE_PROJECT_NAME=$(echo "${BUILD_TAG:-ansible-plugin-testing}-conjur-host-identity" | sed -e 's/[^[:alnum:]]//g' | tr '[:upper:]' '[:lower:]')
```

Comment:
Seeing as this a dev environment, we should hardcode `COMPOSE_PROJECT_NAME` wherever it's used.

Reply:
Done.